ic-os/components/upgrade/shared-resources/upgrade-shared-data-store/upgrade-shared-data-store.service

```diff
@@ -3,6 +3,7 @@ Description=Initialize node data storage
 DefaultDependencies=no
 Requires=var-lib-ic-data.mount
 After=var-lib-ic-data.mount
+Before=setup-permissions.service
```

Comment:
I don't know that these units were competing before, but as these touch the same files, we should be careful with the ordering.